### PR TITLE
add styled table header cell component

### PR DESCRIPTION
**Pull request by Mirmidon**
**What changes did you make?** (Give an overview)
I've added styled table header cell component

**Is there anything you'd like reviewers to focus on?**
overall code style and solution

**How Has This Been Tested?** (put an "X" next to an item)

- [ ] No need to
- [ ] Manually(please, describe, when necessary)
- [ x] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [ ] My changes generate no new warnings(e.g. Sonar is happy)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

File: kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          role="button"
+          tabIndex={0}
+        >
+          {previewText}
+        </span>
+      )}
+      {orderValue && (
+        <span
+          className="icon is-small is-clickable"
+          onClick={() => orderValue && onOrderBy && onOrderBy(orderValue)}
```
Comment (GneyHabub) [inline kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.tsx:40]: I think the whole header should be clickable

File: kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { TopicColumnsToSort } from 'generated-sources';
+import StyledTableHeaderCell from 'components/common/table/TableHeaderCell/TableHeaderCell.styled';
+import cx from 'classnames';
+
+export interface TableHeaderCellProps {
+  title?: string;
+  thType?: 'primary';
```
Comment (GneyHabub) [inline kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.tsx:8]: Why do we need it if there's only one type of header?

File: kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.tsx (hunk 1 above)
Comment (Mirmidon) [reply kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.tsx:40]: in figma there is design for table header cell with "preview" button (I assume it is button) so I separate "preview",  "orderBy" and th element

File: kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.tsx (hunk 2 above)
Comment (Mirmidon) [reply kafka-ui-react-app/src/components/common/table/TableHeaderCell/TableHeaderCell.tsx:8]: in figma there is only one design style, so I've added what we have for now.

**Pull request merged**